tests/modules/eggnog/download/test.yml

```diff
@@ -0,0 +1,12 @@
+- name: eggnog download
+  command: nextflow run ./tests/modules/eggnog/download -entry test_eggnog_download -c tests/config/nextflow.config -stub-run
```

Comment:
Im a bit worried that the tests don't actually run the tool itself. All you are testing here is if the module can create three files and output them. It would not test arguments or that the container ran the tool etc.

Reply:
Totally agree, and you are right. This is just a work around and doesn't actually test the program. I don't think there is anything in place to test modules that require large input databases.